### DISCUSSION TITLE
Owon Bluetooth Devices

### DISCUSSION THREAD

**Discussion by webspiderteam** [webspiderteam/Bluetooth-DMM-For-Windows#40] [Ideas]
@ludwich66 How are you? I wanted to know your thougts about something. 
What do you think about adding Owon dmms protocol?

**Comment by ludwich66** [comment]
> @ludwich66 How are you? I wanted to know your thougts about something.
> What do you think about adding Owon dmms protocol?

Merhaba Motor,
İyiyim, teşekkürler. Şu anda amatör radyo Quansheng UV-K5 için github projesini destekliyorum. Cihaz 20-30 € karşılığında kullanılabilir. Baskılayıcılar donanımın yanı sıra ürün yazılımını da kırar (şema yoktur).
Ve şimdi bebeğimize, OWON cihazınız var mı?
Sadece bir "söküm profesyonelinin" videolarını biliyorum
OW18B: https://www.youtube.com/watch?v=xDyAog1O7Vk&t=689s
GoogleTranslateHelps ;-)
